libraries/StorageManager/StorageManager.cpp

```diff
@@ -35,18 +52,24 @@ extern const AP_HAL::HAL& hal;
   On PX4v1 this gives 309 waypoints, 30 rally points and 52 fence points
   On Pixhawk this gives 724 waypoints, 50 rally points and 84 fence points
  */
-const StorageManager::StorageArea StorageManager::layout_default[STORAGE_NUM_AREAS] = {
```

Comment:
We've lost the sizing here, which I like, but should we have a `static_assert()` here somewhere to ensure the layout is the size we expect?

Reply:
Yes ,we should have a static_assert.I always forget about those ...

Reply:
done !